src/modules/jcms_article/src/Hooks/NodePresave.php

```diff
@@ -138,6 +138,10 @@ public function setSubjectTerms(EntityInterface $entity, ArticleVersions $articl
    * @param \Drupal\jcms_article\Entity\ArticleVersions $article
    */
   public function updateFragmentApi(EntityInterface $entity, ArticleVersions $article) {
+    if (empty(Settings::get('jcms_article_auth_unpublished', FALSE))) {
+      return;
```

Comment:
This will be address in PR: https://github.com/elifesciences/journal-cms/pull/171